### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by riya-kostha**

### CODE REVIEW

File: ArrayList.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+list=[];
+for i in range(71,82):
+    list.append(i)
+
+print("First lis is:",list)
+
+list2=[]
+for i in range(len(list)):
```
Comment (PuspendraSrivastava) [inline ArrayList.py:8]: you don't need to use range here. Can do it like :
for elem in list:

Comment (riya-kostha) [reply ArrayList.py:8]: i did it in another program

File: ArrayList.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+list=[];
+for i in range(71,82):
+    list.append(i)
+
+print("First lis is:",list)
+
+list2=[]
+for i in range(len(list)):
+    list2.append(list.__getitem__(i)*list.__getitem__(i))
```
Comment (PuspendraSrivastava) [inline ArrayList.py:9]: list2.append(elem**2) OR list2.append(elem * elem)

File: Range.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+a = [11,12,13,14,15,16,17,18,19,20]
```
Comment (PuspendraSrivastava) [inline Range.py:1]: you can use range here:
a = list(range(11, 21))

File: ArrayList.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+list=[];
+for i in range(71,82):
```
Comment (PuspendraSrivastava) [inline ArrayList.py:2]: range(71, 81)

File: Range.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+a = [11,12,13,14,15,16,17,18,19,20]
+for i,v in enumerate(a):
+    print("index =",i , "value =" ,v);
```
Comment (PuspendraSrivastava) [inline Range.py:3]: you also need to print type of individual list element:
print("index =",i , "value =" ,v, "Element Type:", type(v))

File: average.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+count=0
+while True:
+        num = input("enter no")
+        if num == 'q':
+            break
+        sum += int(num)
+        count = count+1
+
+
+print("average of numbers is ",int(sum/count))
```
Comment (PuspendraSrivastava) [inline average.py:11]: should not convert average type to int.

File: average.py [hunk 7]
```diff
@@ -0,0 +1,11 @@
+sum = 0
+count=0
+while True:
+        num = input("enter no")
```
Comment (PuspendraSrivastava) [inline average.py:4]: you can probably add in message that 'to quit you can type 'q' otherwise enter number

File: largest.py [hunk 8]
```diff
@@ -0,0 +1,7 @@
+list=[1,2,45,67,89,90,170]
+for i in range(len(list)):
```
Comment (PuspendraSrivastava) [inline largest.py:2]: what is the relevance of this for loop. you can sort a list just by a single statement:
list.sort()

File: largest.py [hunk 9]
```diff
@@ -0,0 +1,7 @@
+list=[1,2,45,67,89,90,170]
+for i in range(len(list)):
+    list.sort()
+
+print("list is",list)
+print("second largest number",list.__getitem__(len(list)-2))
```
Comment (PuspendraSrivastava) [inline largest.py:6]: would be better if you use:
list[len(list)-2] or list[-2]

File: largest.py [hunk 10]
```diff
@@ -0,0 +1,7 @@
+list=[1,2,45,67,89,90,170]
+for i in range(len(list)):
+    list.sort()
+
+print("list is",list)
+print("second largest number",list.__getitem__(len(list)-2))
+print("second smallest number",list.__getitem__(1))
```
Comment (PuspendraSrivastava) [inline largest.py:7]: would be better if you use:
list[1]

File: main.py [hunk 11]
```diff
@@ -0,0 +1,8 @@
+sum = 0
+n = int(input("enter no"))
```
Comment (PuspendraSrivastava) [inline main.py:2]: Here we want sum of only first 100 numbers only. Please make corrections

File: sortList.py [hunk 12]
```diff
@@ -0,0 +1,5 @@
+list = [1,2,389,78,56]
+
+list.sort()
```
Comment (PuspendraSrivastava) [inline sortList.py:3]: you can also write:
list.sort(reverse=True)

then you don't need to reverse it explicitly.

File: NumberOfElement.py [hunk 13]
```diff
@@ -0,0 +1,4 @@
+num=134345
```
Comment (PuspendraSrivastava) [inline NumberOfElement.py:1]: Which assignment is this?

**Review by PuspendraSrivastava — COMMENTED** [review]
Please correct suggested changes

File: average.py (hunk 6 above)
Comment (riya-kostha) [reply average.py:11]: okay

File: largest.py (hunk 8 above)
Comment (riya-kostha) [reply largest.py:2]: yeah it was a mistake there is no neeed of for loop

File: largest.py (hunk 9 above)
Comment (riya-kostha) [reply largest.py:6]: okay

File: NumberOfElement.py (hunk 13 above)
Comment (riya-kostha) [reply NumberOfElement.py:1]: 9

**Comment by riya-kostha** [comment]
> Please correct suggested changes

thanks for your review i will make changes which you suggested

**Pull request merged**